bundles/io.openliberty.tools.eclipse.ui/src/io/openliberty/tools/eclipse/CommandBuilder.java

```diff
@@ -81,6 +81,17 @@ public static String getGradleCommandLine(String projectPath, String cmdArgs, St
         return cmdLine;
     }
 
+
+    public static String getMvnCleanCommand(String projectPath, String pathEnv) throws CommandBuilder.CommandNotFoundException{
+    	try {
+    		CommandBuilder builder = new CommandBuilder(projectPath, pathEnv, true);
+			return builder.getCommand()+" clean && ";
```

Comment:
Rather than adding a separate "mvn clean" appended with && to the dev mode command, we should append "clean" to the list of goals in the single mvn command.  e.g. `mvn clean liberty:dev`